hazelcast-jet-core/src/main/resources/client-protocol-definition/Jet.yaml

```diff
@@ -271,36 +281,3 @@ methods:
           nullable: false
           since: 2.0
           doc: ''
-  - id: 15
-    name: submitLightJob
-    since: 2.3
-    doc: ''
-    request:
-      retryable: true
```

Comment:
In its new state, light job operations are not `retryable`, is this OK? This job submit operation does not look like an idempotent operation and does not tolerate retry (https://github.com/hazelcast/hazelcast/blob/24d3706f8d5725230ffe7399f254cf59ef7e8ac5/hazelcast/src/main/java/com/hazelcast/jet/impl/JobCoordinationService.java#L286), so the current state feels right to me but I just want to have this change confirmed.

Reply:
Hmm, I actually didn't think about it when doing light job. I think it's better if it's not retryable. It could even happen that a job will be executed twice, if the response was lost. So I'd say it was a bug before.